lms/envs/labster.py

```diff
@@ -28,6 +29,7 @@
 )
 
 LABSTER_WIKI_LINK = LABSTER_SETTINGS.get('LABSTER_WIKI_LINK', 'https://theory.labster.com/')
+LABSTER_EDX_AUTH_TOKEN = LABSTER_AUTH.get('LABSTER_EDX_AUTH_TOKEN', '')
```

Comment:
Using this token here limits us to have 2 REGIONS only. While this is correct current situation, very soon we will have more and we will have to fix it. I suggest to put token into https://github.com/Livit/Livit.Learn.EdX/pull/104/files#diff-55b798ee23a7fde8d1103408afcd0f16R1247 REGIONS variable

Reply:
updated @avkoval 
https://github.com/Livit/Livit.Learn.EdxCourseLicenses/pull/16/files#diff-292387c4143e9396a1fd833fa69de7dcR48